services/app-api/handlers/reports/create.ts

```diff
@@ -17,27 +17,29 @@ import {
   reportNames,
 } from "../../utils/constants/constants";
 import { calculatePeriod, convertDateUtcToEt } from "../../utils/time/time";
-import { createReportName } from "../../utils/other/other";
+import {
+  createReportName,
+  getLastCreatedWorkPlan,
+} from "../../utils/other/other";
 // types
 import {
+  AnyObject,
   DynamoWrite,
   isReportType,
   isState,
-  ReportMetadata,
   ReportMetadataShape,
-  ReportStatus,
   ReportType,
   S3Put,
   StatusCodes,
   UserRoles,
 } from "../../utils/types";
 import { getOrCreateFormTemplate } from "../../utils/formTemplates/formTemplates";
 import { logger } from "../../utils/logging";
-import { fetchReport, fetchReportsByState } from "./fetch";
 import { APIGatewayProxyEvent } from "aws-lambda";
 
 export const createReport = handler(
   async (event: APIGatewayProxyEvent, _context) => {
+    // If user who is calling for report creation isn't a state user or state rep, return Unautorized
```

Comment:
if you're gonna make more changes anyway, nit: 'unauthorized'

Reply:
Will do!